test/unit/geo/lng_lat_bounds.test.js

```diff
@@ -81,6 +82,14 @@ test('LngLatBounds', (t) => {
         t.equal(bounds1.getNorth(), 15);
         t.equal(bounds1.getEast(), 15);
 
+        const bounds4 = new LngLatBounds([-20, -20, 20, 20]);
```

Comment:
It would be good to complete test coverage with tests/test cases that
- extend by the various `LngLatLike` inputs 
- check invalid inputs - those that don't match `LngLatLike` or `LngLatBoundsLike`

Reply:
sounds good, will add more test cases.